streams/iq.m3u

```diff
@@ -29,16 +49,46 @@ https://5d94523502c2d.streamlock.net/home/mystream/playlist.m3u8
 http://ns8.indexforce.com:1935/alsharqiyalive/mystream/playlist.m3u8
 #EXTINF:-1 tvg-id="AlSharqiyaNews.iq",Al-Sharqiya News (1080p)
 https://5d94523502c2d.streamlock.net/alsharqiyalive/mystream/playlist.m3u8
+#EXTINF:-1 tvg-id="Alqanat9TV.iq",Alqanat9 TV (1080p)
+https://cdn.bestream.io:19360/alqanat9/alqanat9.m3u8
 #EXTINF:-1 tvg-id="AmozhgaryTV.iq",Amozhgary TV (1080p)
 https://media.streambrothers.com:1936/8248/8248/playlist.m3u8
 #EXTINF:-1 tvg-id="AssyrianANB.iq",Assyrian ANB (720p)
 https://anbsat.ddns.net/anb/live/ANB/playlist.m3u8
-#EXTINF:-1 tvg-id="DijlahTV.iq",Dijlah TV (1080p)
-http://91.134.145.75:10001/Dijlah/index.m3u8
```

Comment:
Works

Reply:
Please add it back

Reply:
> Please add it back

It's an old link and there are always interruptions
The new link I added is much better